src/main/webapp/login.jsp

```diff
@@ -6,55 +6,45 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
 <script src="http://www.w3schools.com/lib/w3data.js"></script>
+<script src="javascript/login.js"></script>
 <body>
-
 <header class="w3-container w3-teal">
     <h1>Login or Register</h1>
 </header>
-
 <div class="w3-container w3-half w3-margin-top">
-
     <form class="w3-container w3-card-4">
         <div id="error-panel" class="w3-panel w3-red w3-hide">
             <h3>Error!</h3>
             <p>{{message}}</p>
         </div>
-
         <p>
             <input id="register-cb" class="w3-check" type="checkbox" onchange="switchRegistration()">
             <label for="register-cb">Registration</label>
         </p>
-
         <p>
             <input id="username-txt" class="w3-input" type="text" style="width:90%" required>
```

Comment:
This style is used for several components and looks like a good candidate for extraction into CSS.

Reply:
There are 6 components on this page that use this style. Please move `style="width:90%"` into `index.css` and refer to the style name instead.